silkworm/capi/silkworm.h

```diff
@@ -120,6 +153,10 @@ struct SilkwormSettings {
     char data_dir_path[SILKWORM_PATH_SIZE];
     //! libmdbx version string in git describe format.
     char libmdbx_version[SILKWORM_GIT_VERSION_SIZE];
+    //! Index salt for block snapshots
+    uint32_t block_index_salt;
+    //! Index salt for state snapshots
+    uint32_t state_index_salt;
```

Comment:
```suggestion
    //! Index salt for block snapshots
    uint32_t blocks_repo_index_salt;
    //! Index salt for state snapshots
    uint32_t state_repo_index_salt;
```
suggest to use a bit more specific name to avoid confusion ("block index" and "state index" might be unclear)